cogs/info.py

```diff
@@ -31,6 +31,29 @@ async def leaderboard(self, ctx: commands.Context):
 
         await ctx.reply(embed=embed)
 
+    @leaderboard.command(aliases=["mg"])
+    async def matchgame(self, ctx: commands.Context, level: str = "1"):
+        """Shows the IUFI Matching Game leaderboard."""
+        users = await func.USERS_DB.find().sort([
+            (f"game_state.match_game.{level}.matched", -1),
+            (f"game_state.match_game.{level}.click_left", -1),
+            (f"game_state.match_game.{level}.finished_time", 1)
+        ]).limit(10).to_list(10)
+
```

Comment:
Add a level check condition, check if that level exists first

Reply:
oh yes i forgot to pull i have that in testing code